### ISSUE TITLE
Accessibility audit - ArticleCard link aria-label

### ISSUE THREAD

**Issue by katarinazaprazna** [bratislava/olo.sk#849]
Aria label articlecards - arialabelledby title
---
accessibility audit PDF:
https://inovaciebratislava.slack.com/archives/C06FW05TXHC/p1730379593029059

![Image](https://github.com/user-attachments/assets/42ede20d-f8aa-46ec-9069-88d0008931e3)
